kubeflow/spark/backends/base_test.py

```
@@ -0,0 +1,255 @@
+            backend.list_sessions()
+        elif method_name == "delete_session":
+            backend.delete_session(name_arg)
+        elif method_name == "_wait_for_session_ready":
+            backend._wait_for_session_ready(name_arg)
+        elif method_name == "get_session_logs":
+            backend.get_session_logs(name_arg)
+
+        assert test_case.expected_status == SUCCESS, (
+            f"Expected exception but none was raised for {test_case.name}"
+        )
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)